kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java

```diff
@@ -24,6 +24,15 @@
   @Parameter(names = "--emit_jvm_signatures", description = "Generate vnames with jvm signatures.")
   private boolean emitJvmSignatures;
 
+  @Parameter(
+      names = "--ignore_typesig_exceptions",
```

Comment:
This seems broadly reasonable to me, but I might like a more descriptive flag name

Reply:
I'm open to suggestions.

Reply:
Do we need to flag this behavior?  It follows the standard Kythe doctrine of letting the indexers "do the best they can" even under error conditions.

Reply:
That works for me, particularly with the error message indicating the workaround.

Reply:
Just to clarify: you propose to remove the flag? If so, will do.

Reply:
Yeah, just remove the flag.

Reply:
Done.